packages/ui-components/src/components/tree-dropdown/tree-dropdown.tsx

```diff
@@ -63,6 +63,7 @@ export class KvTreeDropdown {
 						disabledNodes={this.disabledNodes}
 						highlightedNodes={this.getHighlightedNodes(node)}
 						loadingNodes={this.loadingNodes}
+						showTooltip
```

Comment:
Can we make this configurable but default it to true?

Reply:
Done :heavy_check_mark: